packages/blip/app/components/chart/patientDashboard.js

```diff
@@ -7,12 +7,16 @@ import Header from "./header";
 
 const PatientDashboard = (props) => {
   //eslint-disable-next-line
-  const { patient, prefixURL, profileDialog, bgPrefs, loading, chartPrefs, dataUtil, epochLocation, msRange} = props;
+  const { patient, user, prefixURL, profileDialog, bgPrefs, loading, chartPrefs, dataUtil, epochLocation, msRange, chatWidget: ChatWidget} = props;
   const getEndpoints = () => {
     const start = moment.utc(epochLocation - msRange / 2).toISOString();
     const end = moment.utc(epochLocation + msRange / 2).toISOString();
     return [start, end];
   };
+  /*retrieve for the patient the first monitoring team found (only one monitoring team is allowed)*/
+  // eslint-disable-next-line react/prop-types
```

Comment:
You can remove this eslint comment if you change line 48 by:

```
PatientDashboard.propTypes = {
  user: PropTypes.object,
  chatWidget: PropTypes.object,
  loading: PropTypes.bool.isRequired,
  patient: PropTypes.object,
  prefixURL:  PropTypes.string,
  profileDialog: PropTypes.func,
  bgPrefs: PropTypes.object.isRequired,
  chartPrefs: PropTypes.object.isRequired,
  dataUtil: PropTypes.object,
  epochLocation: PropTypes.number.isRequired,
  msRange: PropTypes.number.isRequired,
};
```
And you should also be able to remove the eslint comment at line 9 and line 41

Reply:
done